### DISCUSSION TITLE
ZepetoPlayers 에서 gravity 값을 어떻게 설정하나요??

### DISCUSSION THREAD

**Discussion by munhyok** [naverz/zepeto-studio-kor#763] [5. Q&A]
전에는 
```
ZepetoCharacterCreator.CreateByZepetoId("id", new SpawnInfo(), (character : ZepetoCharacter)=>{
     
            character.MotionV2.Gravity = 10;
})
```
식으로 연습삼아 만들었는데
이번에 [[zepeto-multiplay-example](https://github.com/naverz/zepeto-multiplay-example)]를 보고 만들어서 멀티플레이는 성공했습니다만
중력값을 어떻게 설정해야할 지 도저히 감이 안잡혀서 여기다가 글 적어봅니다...

근데 예제에는 ZepetoPlayers를 사용하던데
ZepetoPlayers에도 ZepetoPlayers.gravity가 있는것 같더라구요.. 근데 코드를 작성해도 안돼길래..
혹시 좋은 예제가 있다면 알려주시면 감사하겠습니다..!

**Comment by pumfit** [comment]
예시로 주신 코드 처럼  character.MotionV2.Gravity = 1; 로 설정해보니 캐릭터 중력값이 적용되는 것 같습니다. 아래는 사용한 예제코드이고 ZepetoPlayers에서 캐릭터 컨트롤러 버전이 V2로 설정되어 있는지도 확인해보면 좋을 것 같습니다!
```
       ZepetoPlayers.instance.CreatePlayerWithUserId(["userId"], new SpawnInfo(),true);
        
        ZepetoPlayers.instance.OnAddedLocalPlayer.AddListener(()=>{
            ZepetoPlayers.instance.LocalPlayer.zepetoPlayer.character.MotionV2.Gravity = 1;
        });
```
